css/index.css

```diff
@@ -990,16 +1033,73 @@ section {
     }
 }
 
+.foto_reference .foto_reference__title {
+	font-size: 42pt;
+	text-align: center;
+	color: #3ba8d4;
+	margin: 0 0 0.5em 0;
+    line-height: 42pt;
+}
+
+.foto_reference__cards {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+    flex-wrap: wrap;
+	max-width: 1200px;
+    overflow: hidden;
+    max-height: 298px;
+    transition: max-height 1s;
+    padding: 8px 5px 0;
+}
+
+.foto_reference__cards--extended {
+    max-height: 1500px !important;
+}
+
+.foto_reference__show-more {
+    text-align: center;
+    padding: 30px 0 50px;
+}
+
+.foto_reference__show-more__text {
+    color: #3ba8d4;
+    text-decoration: underline;
+    cursor: pointer;
+    font-size: 18pt;
+    font-family: 'Ubuntu', sans-serif;
+}
+
 .objednavka__reference{
     margin: 5em auto;
     display: block;
     width: 90%;
 }
 
-.foto_reference img{
-    width: 90%;
-    margin: auto;
-    display: block;
+.foto_reference img:not(.rozvijime__cards__picture) {
+	width: 90%;
+	margin: auto;
+	display: block;
+}
+
+.foto_reference__cards a:not(.button-fotogalerie):not(.button-video) {
+	width: 33.33%;
```

Comment:
We need to make sure that your spaces match projects space. In our case your editor is set to 4 spaces per tab, but the project is set to 2. Without this, it's hard to read the code without being distracted by the different spaces.

Reply:
It is not complicated in my editor. I will solve this better next time